extension/vpnclient/service_start.sh

```diff
@@ -6,4 +6,4 @@ if [ $# -eq 0 ]; then
   exit 1
 fi
 
-sudo /usr/sbin/openvpn --config "/home/pi/.firewalla/run/ovpn_profile/$1.ovpn" --script-security 2 --route-up "/home/pi/firewalla/extension/vpnclient/ovpn_remove_pushed_routes.sh $1" --up "/home/pi/firewalla/extension/vpnclient/ovpn_up.sh $1" --status "/var/log/openvpn_client-status-$1.log" 2>&1 | sudo tee -a "/var/log/openvpn_client-$1.log"
+sudo /usr/sbin/openvpn --config "/home/pi/.firewalla/run/ovpn_profile/$1.ovpn" --script-security 2 --route-up "/home/pi/firewalla/extension/vpnclient/ovpn_remove_pushed_routes.sh $1" --up "/home/pi/firewalla/extension/vpnclient/ovpn_up.sh $1" --status "/var/log/openvpn_client-status-$1.log" 2>&1 | sudo tee -a "/var/log/openvpn_client-$1.log" | sudo redis-cli HINCRBY "stats:systemd:restart" openvpn 1
```

Comment:
This is vpn client script. No need to monitor this. Need to monitor vpn server instead. You can edit this file
/home/pi/firewalla/vpn/server_route_up.sh